commit 002cc1100ea62f99a4366c9304b9ee76829026b1 (refs/changes/08/1908/1)
Author: David Pacheco <dap@joyent.com>
Date:   2017-05-04T17:02:44-07:00 (2 years, 5 months ago)
    
    joyent/node-fast#6 v2 notes should mention removal of 'after' event

diff --git a/CHANGES.md b/CHANGES.md
index 0ab95e0..6cd84b9 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -62,6 +62,7 @@ Other concrete API differences include:
 * `Client.rpc()` now takes named arguments as mentioned above.
 * `Server.rpc()` is now called `Server.registerRpcMethod()` and takes named
   arguments as mentioned above.
+* The server no longer provides an `after` event after RPC requests complete.
 
 Other notable differences include:
 
